wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Command.java

```diff
@@ -218,6 +218,56 @@ public ParallelRaceGroup onlyWhile(BooleanSupplier condition) {
     return until(() -> !condition.getAsBoolean());
   }
 
+  /**
+   * Decorates this command to run once a condition becomes true.
+   * 
+   * <p>Note: This decorator works by adding this command to a composition. The command the
+   * decorator was called on cannot be scheduled independently or be added to a different
+   * composition (namely, decorators), unless it is manually cleared from the list of composed
+   * commands with {@link CommandScheduler#removeComposedCommand(Command)}. The command composition
+   * returned from this method can be further decorated without issue.
+   * 
+   * @param condition the condition to wait for
+   * @return the decorated command
+   */
+  public SequentialCommandGroup after(BooleanSupplier condition) {
```

Comment:
"after" might be a bit confusing, since we also have decorators that enforce command ordering.

Reply:
I originally had `waitFor` but js suggested `after`, i can see both making sense